hono/test/smoke/outfits.smoke.test.ts

```diff
@@ -63,6 +75,64 @@ describe('[Smoke] Outfits: Seeded [items-simple, outfits-simple]', () => {
   test('GET /outfits: should return 1 seeded outfit with items', async () => {
     const res = await app.request('/api/outfits')
     expect(res.status).toBe(200)
-    expect(await res.json()).toEqual(seededOutfitsSimple())
+    expect(await res.json()).toEqual(
+      testOutfits.map((outfit) => ({
```

Comment:
Might be a good idea to abstract this into the OutfitFactory or ItemToOutfitFactory.

Reply:
For now I think this implementation is fair since it is not portable (yet) to other APIs.